prototype/.editorconfig

```diff
@@ -12,6 +12,8 @@ indent_size = 4
 indent_style = space
 trim_trailing_whitespace = true
 
+[core/models/src/*]
```

Comment:
I think this was added by mistake. It doesn't change any settings.

Reply:
This excludes any checks on the emf stuff. The automatically written ecore and genmodel and the generated code does not conform to the settings of editorconfig. If I don't do it like this, it fails all the time, as the code tends to be regenerated on every build.